Assets/Scripts/Sounds/SoundAssets.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+namespace Sounds
+{
+    public class SoundAssets : MonoBehaviour
+    {
+        [SerializeField] private SoundAudioClip[] soundAudioClipArray;
+
+        public SoundAudioClip[] SoundAudioClipArray => soundAudioClipArray;
+        public static SoundAssets I
```

Comment:
Why the name 'I'?

Reply:
I wanted formulated the question for this place and corrected that. But i forgot this.

Reply:
Fix it now?

Reply:
fixed